alf/experience_replayers/replay_buffer.py

```diff
@@ -134,6 +144,63 @@ def add_batch(self, batch, env_ids=None):
             self._current_size[env_ids] = torch.min(
                 current_size + 1, torch.tensor(self._max_length))
 
+    def dequeue(self, env_ids=None):
+        """Return earliest batch and remove it from buffer."""
```

Comment:
comment argument and return value (shape)

Reply:
Done.